README.md

```diff
@@ -149,6 +149,37 @@ make manifests
 
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
+
+#### Local development on Minikube with internal registry 
+Start the minikube with
+```sh
+minikube start --cpus 4 --memory 4096 --addons enable registry --insecure-registry "localhost:5000"
+```
+In the makefile change
+IMAGE_TAG_BASE to 
+```sh
+IMAGE_TAG_BASE ?= localhost:5000/kiegroup/kogito-serverless-operator
```

Comment:
I don't understand why we need to push the operator into the minikube internal registry.

Don't we only need to know the configurations needed to push the images produced by the operator via kaniko on the internal registry?

Reply:
operator will ned to be available on operatorhub.
however, we still need a doc about running operator locally, but, that can be done with operator-sdk.

Reply:
In general I think that is is better to use the default operator image and explain only to the user to change the DEFAULT_REGISTRY_REPO value into the config map and restart the controller.